std/string.d

```diff
@@ -3889,6 +3916,7 @@ auto rightJustifier(Range)(Range r, size_t width, dchar fillChar = ' ')
 
             void popFront()
             {
+                import std.range.primitives : popFront;
```

Comment:
I'm getting weary of this style, are we promoting this? I mean Importing each range primitive in every scope, if anything I'd expect it to be top level and full.